files/initramfs-tools/varlog

```diff
@@ -11,17 +11,27 @@ case $1 in
     ;;
 esac
 
+logs_inram=false
+
 # Extract kernel parameters
 set -- $(cat /proc/cmdline)
 for x in "$@"; do
     case "$x" in
         varlog_size=*)
             varlog_size="${x#varlog_size=}"
+            ;;
+        logs_inram=on)
+            logs_inram=true
+            ;;
     esac
 done
 
 [ -z "$varlog_size" ] && exit 0
 
+# If logs are being stored in memory, then don't bother
+# creating the log file just to have it deleted afterwards.
+$logs_inram && exit 0
```

Comment:
I've seen this syntax causing trouble when -x was used. Something related to newer version of sh treat unmatched condition as failure. It would be safer to use if statement.
I can see that this syntax was used at line 29 as well.

Not making this issue blocking.

Reply:
Yeah true, I was a bit surprised that it was working here, but for now, I matched the syntax on line 29.